robot-core/src/main/java/org/obolibrary/robot/IOHelper.java

```diff
@@ -27,6 +27,7 @@
 import org.obolibrary.obo2owl.OWLAPIOwl2Obo;
 import org.obolibrary.oboformat.model.OBODoc;
 import org.obolibrary.oboformat.writer.OBOFormatWriter;
+import org.obolibrary.robot.template.TemplateHelper;
```

Comment:
Looks like this isn't used.

Reply:
We have mirrored read methods in `IOHelper` and `TemplateHelper`. (`readCSV`, `readTSV`, `readTable`...). The `IOHelper` methods are never used but removing them would cause a backwards compatibility issue. The `IOHelper` methods just use:
```
return TemplateHelper.readTable(path);
```

Reply:
Ok